pyvrp/cpp/search/Route.h

```diff
@@ -146,7 +151,7 @@ Duration Route::timeWarp() const { return timeWarp_; }
 
 bool Route::empty() const { return size() == 0; }
 
-Load Route::capacity() const { return vehicleType.capacity; }
+Load Route::capacity() const { return data.vehicleType(vehicleType_).capacity; }
```

Comment:
I agree overall this is cleaner (I considered this as well) but we use `Route::capacity()` a lot in `LocalSearch`. I wasn't sure if this additional indexing wouldn't hurt performance? Can we benchmark this or are you confident it doesn't matter?

Reply:
This does the same thing the previous implementation (where `vehicleType` was a const ref) did: look up a memory address. The two are equivalent in that sense. As such, I'm pretty sure this doesn't have an impact on performance.